lib/buildr_plus/features/less.rb

```diff
@@ -17,17 +17,21 @@
     def default_less_path
       'src/main/webapp/less'
     end
+
+    attr_accessor :options
+
+    def less_path
+      options[:source_dir] || default_less_path
+    end
   end
 
   f.enhance(:ProjectExtension) do
     def less_path
-      @less_path || project._(BuildrPlus::Less.default_less_path)
```

Comment:
Why disable overriding per-project? I would be inclined to keep this.

I would also add a per-project options configuraiton